src/components/Layout/Layout.tsx

```diff
@@ -17,12 +16,8 @@ export const Layout: React.FunctionComponent<LayoutProps> & {
   AnnotatedSection: typeof AnnotatedSection;
   Section: typeof Section;
 } = function Layout({sectioned, children}: LayoutProps) {
-  const {newDesignLanguage} = useFeatures();
   const content = sectioned ? <Section>{children}</Section> : children;
-  const className = classNames(
-    styles.Layout,
-    newDesignLanguage && styles.newDesignLanguage,
-  );
+  const className = classNames(styles.Layout);
```

Comment:
You don't need to use the `classNames` function anymore. You can just apply `styles.Layout` on the div below